controllers/actions.github.com/autoscalinglistener_controller.go

```diff
@@ -424,6 +424,35 @@ func (r *AutoscalingListenerReconciler) createListenerPod(ctx context.Context, a
 
 	newPod := r.resourceBuilder.newScaleSetListenerPod(autoscalingListener, serviceAccount, secret, envs...)
 
+	if autoscalingListener.Spec.GitHubServerTLS != nil {
+		var rootCAsConfigMap corev1.ConfigMap
+		err := r.Get(
+			ctx,
+			types.NamespacedName{
+				Namespace: autoscalingRunnerSet.Namespace,
+				Name:      autoscalingListener.Spec.GitHubServerTLS.RootCAsConfigMapRef,
+			},
+			&rootCAsConfigMap,
+		)
+		if err != nil {
+			return ctrl.Result{}, fmt.Errorf(
+				"failed to get configmap %s: %w",
+				autoscalingListener.Spec.GitHubServerTLS.RootCAsConfigMapRef,
+				err,
+			)
+		}
+
+		var certString string
+		for _, cert := range rootCAsConfigMap.Data {
+			certString += cert
+		}
+
+		newPod.Spec.Containers[0].Env = append(newPod.Spec.Containers[0].Env, corev1.EnvVar{
```

Comment:
do we want to move this up before calling `newScaleSetListenerPod` and make it similar to how you handle the proxy env?

Reply:
Pre proxy code, on it 🙌